test/test_skill_resources.py

```diff
@@ -0,0 +1,200 @@
+# NEON AI (TM) SOFTWARE, Software Development Kit & Application Framework
```

Comment:
Direct copy of neon-minerva test class, consider installing/importing rather than duplicating for future maintenance